File: 백준/Silver/3135. 라디오/라디오.java
import java.util.*;
import java.io.*;
import java.lang.reflect.Array;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int a = Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());

        int n  = Integer.parseInt(br.readLine());
        int[] buttons = new int[n];
        for(int i=0;i<n;i++){
            buttons[i] = Integer.parseInt(br.readLine());
        }

        int minClick = Math.abs(b-a);
        for(int button : buttons){
            int click = Math.abs(b-button)+1;
            minClick = Math.min(minClick, click);
        }
        System.out.println(minClick);
    }
}

File: 백준/Silver/3135. 라디오/라디오.py
import sys
input = sys.stdin.readline


a,b = map(int,input().split())
n = int(input())
button = [int(input()) for _ in range(n)]
clicks = abs(b-a)
for x in button:
    clicks = min(clicks,abs(b-x)+1)

print(clicks)